### DISCUSSION TITLE
"LimitExceededException" does not appear in Java SDK but in .NET SDK

### DISCUSSION THREAD

**Discussion by rmt2021** [aws/aws-sdk-java#2843] [Q&A] [answered]
We notice that in the **.NET** SDK, there is a "LimitExceededException" throttling error code which can be retried when throttling happens.
https://github.com/aws/aws-sdk-net/blob/c40919317d59b8fa29e3703ded35c590a13c3490/sdk/src/Core/Amazon.Runtime/Pipeline/RetryHandler/RetryPolicy.cs#L62

However, in the corresponding file RetryUtils.java in the **Java** SDK, there is not "LimitExceededException":

https://github.com/aws/aws-sdk-java/blob/d23ade385264fa8f61b367b2a5c4be132ae49e70/aws-java-sdk-core/src/main/java/com/amazonaws/retry/RetryUtils.java

**Comment by yasminetalby** [comment, marked as answer]
Hello @rmt2021 ,

Thank you very much for your submission and for sharing your observation.
The AWS Java SDK has removed "LimitExceededException" throttling error code in v1.11.753 (see [CHANGELOG-1.11.x.md](https://github.com/aws/aws-sdk-java/blob/master/changelogs/CHANGELOG-1.11.x.md#L6002) )because many services don't treat it as a throttling exception.

Is there any issue you are experiencing caused by the AWS Java SDK approach or do you have any further questions?

Best,

Yasmine
